### ISSUE TITLE
Rework the tagging system when offline

### ISSUE THREAD

**Issue by pixernot** [KeypDemoCorp/awesome-ui#146]
Accessibility should be considered when implementing this.

The design mockups for this are available in the shared drive.

**Comment by pixernot** [comment]
Nice investigation, the root cause analysis is spot on.

I'd vote to close this as a duplicate of the earlier ticket.

**Comment by alachaum** [comment]
Good point, I hadn't considered the offline case.
